shell/pages/c/_cluster/uiplugins/index.vue

```diff
@@ -81,10 +84,13 @@ export default {
       hash.helmOps = await this.$store.dispatch('management/findAll', { type: CATALOG.OPERATION });
     }
 
+    hash.settings = await this.$store.dispatch(`management/findAll`, { type: MANAGEMENT.SETTING });
```

Comment:
@aalves08 We should get version from the new version endpoint - so no need to make this extra fetch - see `config/version.js` - `Version` is a property on the version data - note that on a head build this returns a commit hash.

Reply:
@nwmac fixed the code. Thanks for the heads up 🙏 

On a final build (`vX.X.X`), as per my testing, works perfectly (client's usecase). As for a version that generates a commit hash or any version that return a format different than `vX.X.X`, it "blocks" any extension version that contains a `catalog.cattle.io/ui-version` (check screenshot). I don't feel it's a blocker but let me know if you don't agree.

![Screenshot 2023-02-20 at 17 39 13](https://user-images.githubusercontent.com/97888974/220172345-28a87721-2965-408b-b574-72644f5e5373.png)